templates/sonar_demo.html

```diff
@@ -43,16 +45,18 @@
     // get FFT results and render it on the spectrogram
     fft.event_listeners.push((fft_magnitudes) => {
       if (fft_magnitudes != null) {
-        app.push(fft_magnitudes.slice(395, 415), {z_scale: 5000});
+        //app.push(fft_magnitudes.slice(395, 415), {z_scale: 5000});
+          app.push(fft_magnitudes, {z_scale: 500000});
       }
     });
-
-    const osc = new Tone.Oscillator(19000, 'sine').toDestination();
+    let osc = new Tone.Oscillator(0, 'sine').toDestination();
     let playing = false;
     document.getElementById('play_sound_button').addEventListener('click', (e) => {
       playing = !playing;
-
-      if (playing) {
+        
+      if (playing) { 
+        let freq = document.getElementById("frequency").value;
```

Comment:
- It looks like `freq` should be `const` according to the [style guide](https://google.github.io/styleguide/jsguide.html#features-use-const-and-let) since you're never changging the value of `freq`.
- Use [single quotes](https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes) for string literals.